### DISCUSSION TITLE
Can I set token_ttl to 14 days?

### DISCUSSION THREAD

**Discussion by krajcikondra** [lexik/LexikJWTAuthenticationBundle#979] [General]
Hi, 

I am solving problem with annoying logout. When I login to my webpage application, close browser and after 1 hour open my webpage application again I am logged out and must login again. It`s very annoying.

When I login to facebook (etc.), close browser then I am logged in after a few days. I want it similarly.

Is proper set  token_ttl instead 1 hour (3600) to 14 days?  Or exist some other way how to do it?

Thanks

**Comment by fd6130** [comment]
By default config the token ttl is 1 hour (3600), you can just modify the token_ttl as you like.

And perhaps you need refresh token mechanism, https://github.com/markitosgv/JWTRefreshTokenBundle

**Reply by krajcikondra** [reply]
Ok, thanks.

**Reply by fd6130** [reply]
> Ok, thanks.

Hi, remember to mark as answered if that helped you.
